api/v1beta1/hetznercluster_types.go

```diff
@@ -27,18 +27,18 @@ const (
 	// apiserver.
 	ClusterFinalizer = "hetznercluster.infrastructure.cluster.x-k8s.io"
 	// AllowEmptyControlPlaneAddressAnnotation allows HetznerCluster Webhook
-	// to skip some validation steps for external managed controle planes.
+	// to skip some validation steps for externally managed control planes.
 	AllowEmptyControlPlaneAddressAnnotation = "capi.syself.com/allow-empty-control-plane-address"
 )
 
 // HetznerClusterSpec defines the desired state of HetznerCluster.
 type HetznerClusterSpec struct {
-	// HCloudNetworkSpec defines the Network for Hetzner Cloud. If left empty no private Network is configured.
+	// HCloudNetworkSpec defines details about the Private Network for Hetzner Cloud. If left empty, no private Network is configured.
```

Comment:
I guess we should rather write it without any capital letters to have consistency -> private network